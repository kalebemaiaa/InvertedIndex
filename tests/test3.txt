QueryOr(QueryWord("embaixador"), QueryAndS(QueryWord("secretário"), QueryWord("chanceler")))
1804.raw
8774.raw
5662.raw
567.raw
3210.raw
1955.raw
2974.raw
5008.raw
980.raw
526.raw
5657.raw
396.raw
4753.raw
5170.raw
3444.raw
11825.raw
1243.raw
2039.raw
1451.raw
11101.raw
4134.raw
5299.raw
3859.raw
1112.raw
539.raw
2164.raw
11092.raw
1104.raw
4173.raw
3371.raw
3565.raw
8717.raw
6172.raw
3777.raw
11844.raw
1109.raw
3601.raw
11261.raw
6162.raw
1213.raw
11176.raw
4253.raw
11171.raw
4851.raw
624.raw
1580.raw
2831.raw
1156.raw
8887.raw
663.raw
4061.raw
1663.raw
4132.raw
425.raw
10964.raw
4436.raw
11275.raw
2946.raw
4719.raw
4169.raw
4029.raw
3713.raw
2184.raw
8835.raw
1772.raw
1351.raw
4810.raw
5154.raw
2168.raw
1607.raw
1807.raw
2773.raw
11641.raw
4825.raw
4630.raw
2063.raw
3227.raw
2183.raw
8793.raw
5419.raw
8842.raw
11205.raw
1067.raw
3235.raw
445.raw
4012.raw
11412.raw
11210.raw
3825.raw
1418.raw
3588.raw
11655.raw
8824.raw
2426.raw
6380.raw
11178.raw
6368.raw
165.raw
2579.raw
1953.raw
11847.raw
3086.raw
6031.raw
3867.raw
506.raw
2082.raw
4378.raw
5835.raw
4694.raw
3761.raw
1056.raw
4798.raw
5909.raw
3951.raw
2059.raw
8812.raw
4764.raw
11439.raw
8867.raw
636.raw
11670.raw
18.raw
11526.raw
659.raw
11464.raw
5348.raw
3941.raw
1183.raw
1858.raw
1929.raw
4327.raw
1399.raw
5164.raw
2380.raw
1346.raw
4941.raw
4546.raw
6174.raw
1796.raw
3584.raw
3459.raw
170.raw
773.raw
787.raw
6123.raw
11045.raw
11354.raw
4547.raw
2978.raw
60.raw
1890.raw
10997.raw
1024.raw
2311.raw
5564.raw
11640.raw
4568.raw
1780.raw
11875.raw
534.raw
3112.raw
393.raw
5075.raw
3007.raw
3998.raw
107.raw
1150.raw
11225.raw
8997.raw
6397.raw
4968.raw
762.raw
11783.raw
3884.raw
10937.raw
6038.raw
2576.raw
416.raw
3936.raw
11775.raw
3023.raw
11615.raw
1098.raw
2124.raw
2541.raw
727.raw
8822.raw
5754.raw
6230.raw
10965.raw
726.raw
2491.raw
1394.raw
4901.raw
1689.raw
4790.raw
11433.raw
2460.raw
5632.raw
3990.raw
