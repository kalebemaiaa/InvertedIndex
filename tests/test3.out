902
954
1013
1150
1246
1601
2966
3163
3838
4111
4730
5433
5545
11098
